src/z3c/autoinclude/dependency.txt

```diff
@@ -86,8 +86,8 @@ that it depends on that have ``configure.zcml`` or ``meta.zcml``.
 Note that the individual lists within ``includableInfo`` preserve the
 package order defined in ``setup.py``::
 
-    >>> a_include_finder.includableInfo(['configure.zcml', 'meta.zcml'])
-    {'configure.zcml': ['b.c'], 'meta.zcml': ['z3c.autoinclude', 'testdirective']}
+    >>> list(sorted(list(a_include_finder.includableInfo(['configure.zcml', 'meta.zcml']).items())))
```

Comment:
You can simplify that to `list(sorted(a_include_finder....items()))` -- sorted() accepts any iterable.

Perhaps a slightly more readable alternative would be to `pprint.pprint(...)` the dict.